web/profiles/custom/yalesites_profile/config/sync/block.block.social_links_block.yml

```diff
@@ -0,0 +1,20 @@
+uuid: a349192a-9081-41c1-8662-83b576fbf15c
```

Comment:
We exported this block config. Still need to figure out the config split/ignore/whatever thing.